apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -0,0 +1,335 @@
+    this.outputData.controlSets = []
+  }
+
+   copyComplianceCards(exportWindow: Window) {
+    // Grab the html from Heimdall, and then grab the container that the html will be put into
+    const statusCardHTML = document.getElementById('statusCounts')?.innerHTML
+    const statusCountCardContainer = exportWindow.document.getElementById('statusCountCard')
+    const severityCardHTML = document.getElementById('severityCounts')?.innerHTML
+    const severityCardContainer = exportWindow.document.getElementById('severityCard')
+    const complianceCardHTML = document.getElementById('complianceLevel')?.innerHTML
+    const complianceCardcontainer = exportWindow.document.getElementById('complianceCard')
+    if(statusCardHTML && severityCardHTML && complianceCardHTML) {
+      // Append the HTML to each container
+      statusCountCardContainer?.insertAdjacentHTML('afterbegin', statusCardHTML)
+      severityCardContainer?.insertAdjacentHTML('afterbegin', severityCardHTML)
+      complianceCardcontainer?.insertAdjacentHTML('afterbegin', complianceCardHTML)
+      // There seems to be some sort of automatic formatting applied with .resize-triggers that doesn't apply here
+      const elements = exportWindow.document.getElementsByClassName('resize-triggers');
+      while (elements[0]) {
+        const element = elements[0];
+        element?.parentNode?.removeChild(element);
+      }
+    }
+  }
+
+    addDetails(control: ContextualizedControl): ContextualizedControl & {details: Detail[]} {
+      return {...control, full_code: control.full_code, details: [
+      {
+        name: 'Control',
+        value: control.data.id
+      },
+      {
+        name: 'Title',
+        value: control.data.title
+      },
+      {
+        name: 'Caveat',
+        value: control.hdf.descriptions.caveat
+      },
+      {
+        name: 'Desc',
+        value: control.data.desc
+      },
+      {
+        name: 'Rationale',
+        value: control.hdf.descriptions.rationale
+      },
+      {
+        name: 'Justification',
+        value: control.hdf.descriptions.justification
+      },
+      {
+        name: 'Severity',
+        value: control.root.hdf.severity
+      },
+      {
+        name: 'Impact',
+        value: control.data.impact
+      },
+      {
+        name: 'Nist controls',
+        value: control.hdf.raw_nist_tags.join(', ')
+      },
+      {
+        name: 'Check Text',
+        value: control.hdf.descriptions.check || control.data.tags.check
+      },
+      {
+        name: 'Fix Text',
+        value: control.hdf.descriptions.fix || control.data.tags.fix
+      }
+    ].filter((v) => v.value)}
+  }
+
+  /** Converts a string to an array buffer */
+  s2ab(s: string) {
+		const buf = new ArrayBuffer(s.length); //convert s to arrayBuffer
+		const view = new Uint8Array(buf); //create uint8array as viewer
+		for(let i = 0; i < s.length; i++) {
+			view[i] = s.charCodeAt(i) & 0xff; //convert to octet
+		}
+		return buf;
+	}
```

Comment:
This function seems to be duplicated in every Export function, could it get de-duplicated and moved into the `utilities` folder instead?